ext/gd/tests/imagecolorclosesthwb_basic_001.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Test imagecolorclosesthwb() basic functionality
+--CREDITS--
+Rodrigo Prado de Jesus <royopa [at] gmail [dot] com>
+#PHPSP User Group Brazil
+--SKIPIF--
+<?php
+	if (!extension_loaded('gd')) {
+		die("skip gd extension not available.");
+	}
+?>
+--FILE--
+<?php
+	$cwd = dirname(__FILE__);
+	$image = imagecreatefromjpeg($cwd . "/conv_test.jpeg");
+	var_dump(imagecolorclosesthwb($image, 116, 115, 152));
+	imagedestroy($image);
+?>
+--EXPECTF--
+int(%i)
```

Comment:
This shouldn't use a wildcard, the result of this function should be deterministic.

Reply:
Done @nikic